libmcount/mcount.c

```diff
@@ -1143,6 +1143,9 @@ void mcount_exit_filter_record(struct mcount_thread_data *mtdp, struct mcount_re
 {
 	uint64_t time_filter = mtdp->filter.time;
 
+	if (time_filter == 0)
+		time_filter = mcount_threshold;
```

Comment:
This breaks the test 113 (trigger_time).  But it was because the test uses `time=0` which disables the time filter and trace all child functions.  The intention was to not trace any child funcitons so I think we need to change it to `time=1`.

Maybe you'd better to update the documentation too.

Reply:
That'd be better!

Reply:
Cool. I'm adding a commit to prevent the user from using `max(uint64)` as time filter.

Reply:
Looks ok.